client/src/components/examples/TagInputContainer.tsx

```diff
@@ -2,41 +2,41 @@ import React, { FunctionComponent, useState } from "react";
 import { TagInput } from "../atoms/TagInput";
 
```

Comment:
This file is just the example usage for `TagInput`, so you can ignore these changes.